strawberryfields/program.py

```diff
@@ -772,49 +702,90 @@ def compile(self, backend='fock', optimize=True):
         Returns:
             Program: compiled program
         """
-        db = backend_database[backend]
+        # TODO: Avoid circular import
+        from strawberryfields.data import backend_databases
```

Comment:
I think circular imports can be avoided if you use "import X" instead of "from X import Y".

Reply:
I tried `import strawberryfields.data` but no luck. From what I can tell, the issue is that `data.chip0` imports `io`, which then imports `Program`.

Reply:
Fixed in the latest commit